netci.groovy

```diff
@@ -321,6 +333,16 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
     // Pull request builds.  Generally these fall into two categories: default triggers and on-demand triggers
     // We generally only have a distinct set of default triggers but a bunch of on-demand ones.
     def osGroup = getOSGroup(os)
+
+    // Get crossgen out of the way
+    if (doCrossgen) {
+        assert (os == 'Windows_NT' || os in Constants.crosslist)
+        // Add trigger phrase for Checked x64 pri1 Crossgen builds
+        if (configuration == 'Checked' && architecture == 'x64' && scenario == 'pri1') {
```

Comment:
See above for the "Checked" condition.

Also, is Pri1 run enabled for per commit or would need to be invoked via phrase?